### PR TITLE
OCLOMRS-451: The remove button on the name and description column of creating new concept does not work

### PR DESCRIPTION
**Pull request by judeinno**
# JIRA TICKET NAME:
[The remove button on the name and description column of creating new concept does not work](https://issues.openmrs.org/browse/OCLOMRS-451)

# Summary:
On the create new concept page:
- The button for remove name does not function as expected. 
- Also the remove description button, not function as expected.

### CODE REVIEW

**Comment by AGMETEOR** [comment]
but fix the drop in coverage...

**Comment by dkayiwa** [comment]
Any reason why this is still open for a full week?

**Comment by judeinno** [comment]
Yes, you had requested for the remove labels to be changed to buttons
before you can merge it. And the PR is up for changing the buttons just
waiting for all 5 approval then request for merging.
After that then I can request for merging this one again.

File: src/components/dictionaryConcepts/components/DescriptionRow.jsx [hunk 1]
```diff
@@ -124,7 +126,9 @@ class DescriptionRow extends Component {
             className=" btn btn-danger concept-form-table-link"
             id="remove-description"
             type="button"
-            onClick={event => this.handleRemove(event, this.props.newRow)}
+            onClick={event => ((this.props.newRowUuid === '')
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/DescriptionRow.jsx]: Why not instead check for null?

Comment (judeinno) [reply src/components/dictionaryConcepts/components/DescriptionRow.jsx]: Because the value am checking against is a string and null is of object type

Comment (dkayiwa) [reply src/components/dictionaryConcepts/components/DescriptionRow.jsx]: What is the difference between newRow and newRowUuid?

Comment (judeinno) [reply src/components/dictionaryConcepts/components/DescriptionRow.jsx]: newRow is used on the Edit concept interface to access a specific description row details and newRowUuid is used when on the create concepts page when accessing a specific description row id that is stored in a list called description.

Comment (judeinno) [reply src/components/dictionaryConcepts/components/DescriptionRow.jsx]: The Implementaion for the adding of a description row on the edit concept page and the create concept page are different. This is where the two different terms arise from.
When adding a new decription row on the edit concept page.
- This stores the row data in state as an object with key as Uuid so when removing the row we access the id as newRow.uuid

while
When adding a new decription row on the create concept page.
- This stores the row data in state as a list of UUIDs in form of strings so when removing the row we access the id directly

Hence the two different term

**Pull request merged**